File: UAV model/setAutopilotGains.cpp
#ifndef auto
#define auto

struct autoPilot
{
    float kVP = 1/20;
    float kVI = 0.1;

    float khP = 0.7;
    float khI = 0.03;

    float kthetaP = 0.8;
    float kthetaI = 0.03;

    float kqP = 0.8;

    float kpsiP = 0.4;
    float kpsiI = 0.3;

    float kphiP = 5;
    float kphiI = .0;

    float kpP = 0.75;
};

#endif // autoPilot

File: UAV model/UAV model/setAutopilotGains.m
%% Set Autopilot Gains

global pid

pid.kVP = 1/20;
pid.kVI = 0.1;

pid.khP = 0.7;
pid.khI = 0.03;

pid.kthetaP = 0.8;
pid.kthetaI = 0.03;

pid.kqP = 0.8;

pid.kpsiP = 0.4;
pid.kpsiI = 0.3;

pid.kphiP = 5;
pid.kphiI = .0;

pid.kpP = 0.75;